lib/minimax.rb

```diff
@@ -0,0 +1,49 @@
+class Minimax
+
+  def find_best_move(board, depth=0, best_score={}, current_player, player1, player2)
```

Comment:
Are these the marks for the current and other players? Do you need to pass current player and both player one and player two?

Do the depth and best score need to be method parameters?

Reply:
Good point regarding passing in the players. I'll look into this :)
Best score definitely does not need to be a method parameter. 
I'll look into keeping track of depth, without keeping it as a method parameter. Thanks!